server/src/controller/master.controller.ts

```diff
@@ -117,17 +120,13 @@ export const getAvailableMasters = async (req: Request, res: Response) => {
 export const putMaster = async (req: Request, res: Response) => {
         
     try {
-        const { id, name, cities_id } = req.body
-        
-        const updateMaster = await db.query('UPDATE masters SET name = $2 WHERE id = $1', [id, name])
+        const { id, name, citiesId } = req.body
 
-        const masterToCitiesValues = cities_id.map(function(elem: number) {return `(${id},${elem})`}).join()
-
-        const deleteOldData = await db.query('DELETE FROM masters_cities WHERE master_id = $1', [id])
-
-        const updateCitiesOfMaster = await db.query(`INSERT INTO masters_cities (master_id, city_id) VALUES ${masterToCitiesValues}`)
+        const [rows, updateMaster] = await db.Master.update({ name }, {where:{ id }, returning: true})
         
-        res.status(200).json(updateMaster.rows)
+        const updateCitiesOfMaster = await updateMaster[0].setCities(citiesId)
```

Comment:
updateMaster can be []. Please check the name of variables! Why are you using "updateMaster" instead of master?